enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -67,6 +67,11 @@ const (
 	// backwards-compatible changes to image cache storage, and older version
 	// directories can be cleaned up.
 	imageCacheVersion = "v1" // TODO: add automatic cleanup if this is bumped.
+
+	// The kernel's page size determines the maximum length of the "mount" syscall's options string.
+	// Actual value could be different on different systems, but 4096 is a common value.
+	// Use "getconf PAGE_SIZE" to check.
+	kernelPageSize = 4096
```

Comment:
in my testing, the mount options string has to be <= 4095 chars long (4096 results in an error)

Also, I'm not 100% sure if the limit is actually related to page size or if it's just arbitrarily hard-coded somewhere to be 4K. We could check the kernel source to verify, or for now just call this `mountOptionsLengthLimit = 4095` and avoid mentioning page size for now.

Reply:
changed this to `maxMntOptsLength` to avoid mentioning the kernel's page size.